cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -801,28 +801,30 @@ public AppBannerPopup getAppBannerPopup(Banner banner) {
     return new AppBannerPopup(getCurrentActivity(), banner);
   }
 
-  void showBanner(AppBannerPopup bannerPopup) {
+  void showBanner(AppBannerPopup bannerPopup, boolean force) {
```

Comment:
do we want to do boolean force = false?
then we dont have to pass false everywhere
@gbhrdt

Reply:
yea sounds good, but I think in Java it's not possible to give default values to arguments in the method signature - so we would have to create another signature without the `force` argument and handle the default there? :-)

Reply:
So do I have to create a new method without a force variable? 
And also need to change in ios as well.

Reply:
yes please create another overloading method so that we dont need to pass false everywhere, it will be different to ios in this case buts its okay Marius said!